src/gsAssembler/gsAssemblerOptions.h

```diff
@@ -284,8 +284,8 @@ struct gsAssemblerOptions
     // integral computations will be set to the integer which is
     // closest to (A * p + B), where \a p is the (coordinate-wise)
     // degree of the basis
-    double quA;
-    int    quB;
+    double  quA;
+    index_t quB;
```

Comment:
This should be either int or short_t. I would prefer int.
It is not related to index_t.